include/fmt/color.h

```diff
@@ -270,54 +424,34 @@ inline void reset_color<wchar_t>(FILE *stream) FMT_NOEXCEPT {
 
 template <
   typename S, typename Char = typename internal::char_t<S>::type>
-void vprint_rgb(rgb fd, const S &format,
-                basic_format_args<typename buffer_context<Char>::type> args) {
-  internal::fputs<Char>(internal::make_foreground_color<Char>(fd), stdout);
-  vprint(format, args);
-  internal::reset_color<Char>(stdout);
-}
-
-template <
-  typename S, typename Char = typename internal::char_t<S>::type>
-void vprint_rgb(rgb fd, rgb bg, const S &format,
+void vprint_text_format(const text_format &tf, const S &format,
                 basic_format_args<typename buffer_context<Char>::type> args) {
-  internal::fputs<Char>(internal::make_foreground_color<Char>(fd), stdout);
-  internal::fputs<Char>(internal::make_background_color<Char>(bg), stdout);
+  internal::fputs<Char>(internal::make_emphasis<Char>(tf.get_emphasis()), stdout);
```

Comment:
Shouldn't we only do this when emphasis is set?

Reply:
That gets handled elsewhere (`ansi_color_escape`). I can also handle it here if you'd like.

Reply:
I'd suggest omitting the call to `fputs` altogether if emphasis is not set.